runtime/src/lib.rs

```diff
@@ -303,31 +330,38 @@ impl pallet_balances::Config for Runtime {
 	type ExistentialDeposit = ExistentialDeposit;
 	type AccountStore = System;
 	type WeightInfo = pallet_balances::weights::SubstrateWeight<Runtime>;
-}
-
-parameter_types! {
-	/// 1 CAN = 1 Megabyte
-	pub const TransactionByteFee: Balance = CANS / (1024 * 1024);
+	type MaxLocks = MaxLocks;
+	type MaxReserves = MaxReserves;
+	type ReserveIdentifier = [u8; 8];
 }
 
 impl pallet_transaction_payment::Config for Runtime {
-	type OnChargeTransaction = CurrencyAdapter<Balances, ()>;
+	type OnChargeTransaction = pallet_transaction_payment::CurrencyAdapter<Balances, ()>;
 	type TransactionByteFee = TransactionByteFee;
 	type WeightToFee = IdentityFee<Balance>;
 	type FeeMultiplierUpdate = ();
 }
 
+impl pallet_randomness_collective_flip::Config for Runtime {}
+
+impl pallet_sudo::Config for Runtime {
+	type Call = Call;
+	type Event = Event;
+}
+
 parameter_types! {
-	pub TombstoneDeposit: Balance = deposit(
-		1,
-		<pallet_contracts::Pallet<Runtime>>::contract_info_size(),
-	);
+	// NOTE: These are all set to `0` since we're planning on getting rid
+	// of state-rent with an upcoming Substrate release.
+	//
+	// See: https://github.com/paritytech/substrate/pull/9669
+	pub TombstoneDeposit: Balance = TombstoneDeposit::get();
 	pub DepositPerContract: Balance = TombstoneDeposit::get();
```

Comment:
```suggestion
	pub TombstoneDeposit: Balance = deposit(
		1,
		<pallet_contracts::Pallet<Runtime>>::contract_info_size(),
	);
	
	// NOTE: These are all set to `0` since we're planning on getting rid
	// of state-rent with an upcoming Substrate release.
	//
	// See: https://github.com/paritytech/substrate/pull/9669
	pub DepositPerContract: Balance = 0;
```